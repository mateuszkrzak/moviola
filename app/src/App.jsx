import React from 'react';

import 'typeface-roboto'; // eslint-disable-line
import './App.scss';
import Footer from './common/components/Footer/Footer';
import ErrorBoundary from './common/components/ErrorBoundary/ErrorBoundary';
import SearchPage from './containers/SearchPage/SearchPage';
import MovieDetailsPage from './containers/MovieDetailsPage/MovieDetailsPage';

const App = () => (
  <section styleName="wrapper">
    <ErrorBoundary>
      {/* <SearchPage /> */}
      <MovieDetailsPage />
      <Footer />
    </ErrorBoundary>
  </section>
);

export default App;
