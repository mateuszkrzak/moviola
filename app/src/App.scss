@import "./common/variables.scss";

html,
body {
  width: 100%;
  height: 100%;
}

.wrapper {
  min-height: 100vh;
  display: grid;
  grid-template-rows: 1fr auto;
  grid-template-columns: 100%;
  background-color: #dadada;
}
