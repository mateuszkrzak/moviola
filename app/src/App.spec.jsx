import { shallow } from 'enzyme';
import React from 'react';
import App from './App';

test('App component renders search page by default', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
