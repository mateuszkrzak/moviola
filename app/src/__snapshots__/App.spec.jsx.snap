// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders search page by default 1`] = `
<section
  styleName="wrapper"
>
  <ErrorBoundary>
    <SearchPage />
    <Footer />
  </ErrorBoundary>
</section>
`;
